com.unity.netcode.gameobjects/Runtime/Simulator/NetworkTypePresets.cs

```diff
@@ -8,19 +8,19 @@ public class NetworkTypePresets
-        public static readonly NetworkTypeConfiguration Mobile3_75G = NetworkTypeConfiguration.Create("Mobile 3.75G [HDSDPA+, H+, '11]", k_DecentMobileDescription, 75, 50, 5, 0, 0, 0, 0);
-        public static readonly NetworkTypeConfiguration Mobile4G = NetworkTypeConfiguration.Create("Mobile 4G [4G, LTE, '13]", k_DecentMobileDescription, 50, 25, 3, 0, 0, 0, 0);
-        public static readonly NetworkTypeConfiguration Mobile4_5G = NetworkTypeConfiguration.Create("Mobile 4.5G [4G+, LTE-A, '16]", k_DecentMobileDescription, 50, 25, 3, 0, 0, 0, 0);
-        public static readonly NetworkTypeConfiguration Mobile5G = NetworkTypeConfiguration.Create("Mobile 5G ['20]", k_GoodMobileDescription, 1, 10, 1, 0, 0, 0, 0);
+        public static readonly NetworkSimulationConfiguration None = NetworkSimulationConfiguration.Create("None", string.Empty, 0, 0, 0, 0, 0, 0, 0);
+        public static readonly NetworkSimulationConfiguration HomeBroadband = NetworkSimulationConfiguration.Create("Home Broadband [WIFI, Cable, Console, PC]", k_BroadbandDescription, 2, 2, 1, 0, 0, 0, 0);
+        public static readonly NetworkSimulationConfiguration Mobile2G = NetworkSimulationConfiguration.Create("Mobile 2G [CDMA & GSM, '00]", k_PoorMobileDescription, 400, 200, 5, 0, 0, 0, 0);
+        public static readonly NetworkSimulationConfiguration Mobile2_5G = NetworkSimulationConfiguration.Create("Mobile 2.5G [GPRS, G, '00]", k_PoorMobileDescription, 200, 100, 5, 0, 0, 0, 0);
+        public static readonly NetworkSimulationConfiguration Mobile2_75G = NetworkSimulationConfiguration.Create("Mobile 2.75G [Edge, E, '06]", k_PoorMobileDescription, 200, 100, 5, 0, 0, 0, 0);
+        public static readonly NetworkSimulationConfiguration Mobile3G = NetworkSimulationConfiguration.Create("Mobile 3G [WCDMA & UMTS, '03]", k_PoorMobileDescription, 200, 100, 5, 0, 0, 0, 0);
+        public static readonly NetworkSimulationConfiguration Mobile3_5G = NetworkSimulationConfiguration.Create("Mobile 3.5G [HSDPA, H, '06]", k_MediumMobileDescription, 75, 50, 5, 0, 0, 0, 0);
+        public static readonly NetworkSimulationConfiguration Mobile3_75G = NetworkSimulationConfiguration.Create("Mobile 3.75G [HDSDPA+, H+, '11]", k_DecentMobileDescription, 75, 50, 5, 0, 0, 0, 0);
+        public static readonly NetworkSimulationConfiguration Mobile4G = NetworkSimulationConfiguration.Create("Mobile 4G [4G, LTE, '13]", k_DecentMobileDescription, 50, 25, 3, 0, 0, 0, 0);
+        public static readonly NetworkSimulationConfiguration Mobile4_5G = NetworkSimulationConfiguration.Create("Mobile 4.5G [4G+, LTE-A, '16]", k_DecentMobileDescription, 50, 25, 3, 0, 0, 0, 0);
+        public static readonly NetworkSimulationConfiguration Mobile5G = NetworkSimulationConfiguration.Create("Mobile 5G ['20]", k_GoodMobileDescription, 1, 10, 1, 0, 0, 0, 0);
```

Comment:
I think this would be more readable if these declarations were split over multiple lines and using named parameters.